### PR TITLE
Merge template updates

### PR DESCRIPTION
**Pull request by jonasscheid**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mhcquant _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.6.0dev - nfcore/mhcquant
+## v2.6.0dev - [date]
```
Comment (mapo9) [inline CHANGELOG.md:6]: dont you need to add the date here?

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ First, prepare a samplesheet with your input data that looks as follows:
 
 `samplesheet.tsv`:
 
-```tsv
+```csv title="samplesheet.tsv
```
Comment (mapo9) [inline README.md:39]:
```suggestion
```tsv title="samplesheet.tsv
```

File: docs/usage.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ You will need to create a samplesheet with information about the samples you wou
 The `sample` identifiers have to be the same when you have multiple runs. The `Condition` identifiers can be used to further distinguish the sample groups.
 Below is an example for the same sample sequenced across 3 lanes:
 
-```tsv
+```csv title="samplesheet.tsv
```
Comment (mapo9) [inline docs/usage.md:20]:
```suggestion
```tsv title="samplesheet.tsv
```

**Review by mapo9 — CHANGES_REQUESTED** [review]
LGTM! Minor changes maybe?

File: CHANGELOG.md (hunk 1 above)
Comment (jonasscheid) [reply CHANGELOG.md:6]: I think its a placeholder, see e.g. https://github.com/nf-core/rnaseq/blob/dev/CHANGELOG.md#v3140dev---date

**Review by mapo9 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**